### PR TITLE
CICD-295 a

### PR DESCRIPTION
**Pull request by cdwbrad**
## Checklist

- [] I have read the [policy contribution guidelines](https://github.com/kyverno/policies/blob/main/README.md#contribution).
- [] I have added test manifests and resources covering both positive and negative tests that prove this policy works as intended.
- [] I have added the artifacthub-pkg.yml file and have verified it is complete and correct.

### CODE REVIEW

**Review by cdwshawn — CHANGES_REQUESTED** [review]
The following files need remediation based on output from `shellecheck`:

```
cdw/mans/best-practices/overlays/eval/dedupe-policies.sh
cdw/mans/best-practices/overlays/prod/dedupe-policies.sh
cdw/mans/best-practices/overlays/stage/dedupe-policies.sh
```

File: cdw/mans/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+### How things are organized
+Kustomize provides a solution for customizing Kubernetes resource configuration
+free from templates and DSLs. It lets you customize raw, template-free YAML files
+for multiple purposes, leaving the original YAML untouched and usable as is.
+
+With this in mind, the simplest approach to understanding "how things work" is
+to treat `kustomization.yaml` files as bread crumbs. That is to say, where you
+find a kustomization.yaml file, it will take you to your next
+kustomization.yaml...which in turn will take you to another
+kustomization.yaml...etc., etc.
```
Comment (cdwshawn) [inline cdw/mans/README.md:15]: Backtick `kustomization.yaml` for consistency.

**Review by cdwshawn — CHANGES_REQUESTED** [review]

File: cdw/mans/README.md (hunk 1 above)
Comment (cdwbrad) [reply cdw/mans/README.md]: done

**Comment by cdwbrad** [comment]
> 

ran scripts thru shellcheck and remediated.

**Pull request merged**